data/migrations/v0.9.0-polygons.sql

```diff
@@ -4,6 +4,13 @@ UPDATE planet_osm_polygon SET
     "disused" <> 'no' OR
     "railway" = 'station';
 
+UPDATE planet_osm_polygon SET
```

Comment:
Does this need to have the same `ALTER TABLE` as `apply-planet_osm_polygon.sql` to add the column?